src/Console/IndexCommand.php

```diff
@@ -123,4 +132,39 @@ protected function getCount()
     {
         return count($this->getData());
     }
+
+    /**
+     * @param array $responses
+     * @return array
+     */
+    protected function getErrors($responses) 
```

Comment:
I would move all this to a separate class, maybe `BulkResponseAggregator`, that way you can add some unit tests for it.

Reply:
Any particular generator in mind? `Nord\Lumen\Elasticsearch\Search\Aggregation\AggregationCollection` seems reusable.

Reply:
I would just make a new class